### PR TITLE
styling changes to about us cards

### PR DESCRIPTION
**Pull request by JJSanch**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -49,6 +49,20 @@
...
+
+.about-cards{
+  width: 40%;
+  text-align: center;
+  vertical-align: center;
+  background-color: whitesmoke;
+  margin: 1% 40% 10% 30%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  
+  }
```
Comment (minachuong) [inline app/assets/stylesheets/application.scss]:
```suggestion
}
```
indentation of curly braces should align with where the selector is provided

File: app/javascript/components/pages/AboutUs.js [hunk 2]
```diff
@@ -12,8 +12,8 @@ import {
 
 const AboutUs = (props) => {
   return (
-    <div>
-      <Card>
+    <div className="div-container">
```
Comment (minachuong) [inline app/javascript/components/pages/AboutUs.js:15]: I believe reactstrap provides out of the box containers you can use like `<Col>` and `<Row>`

**Review by minachuong — APPROVED** [review]
👍 please see comments

File: app/javascript/components/pages/AboutUs.js (hunk 2 above)
Comment (JJSanch) [reply app/javascript/components/pages/AboutUs.js:15]: Got it

**Pull request merged**